code/datums/spells/horsemask.dm

```diff
@@ -26,7 +26,8 @@
 	if(!target)
 		return
 
-	if((target.type in compatible_mobs) || ishuman(target))
+
+	if(!is_type_in_list(target, compatible_mobs))

```

Comment:
That would be a species check though, not a mob check, which is completely different.

And again, we're not going to have a mob that can equip clothing that isn't human remotely any time soon.

`is_type_in_list` is fairly expensive--`ishuman` is not.